packages/ui-extensions-react/src/surfaces/checkout/hooks/api.ts

```diff
@@ -26,9 +31,13 @@ export function useApi<
 }
 
 /**
- * Returns the full API object that was passed in to your
- * extension when it was created.
- * @deprecated you shoud be importing useApi instead
```

Comment:
Was this removed on purpose?

Reply:
Yes, plan is to not mark these as deprecated yet for `2023-04`. `2023-07` will introduce deprecations.